crates/pallet-domains/src/lib.rs

```diff
@@ -2466,6 +2569,11 @@ impl<T: Config> Pallet<T> {
         )
     }
 
+    pub fn max_prune_domain_execution_receipt() -> Weight {
+        T::WeightInfo::handle_bad_receipt(MAX_BUNLDE_PER_BLOCK)
+            .saturating_add(T::DbWeight::get().reads_writes(3, 1))
+    }
```

Comment:
The extrinsic weights are not just the database write & read but also a base weight, thus would prefer to have proper benchmarking like `submit_fraud_proof`, though non-blocker for this PR.